main.go

```diff
@@ -106,12 +107,13 @@ func main() {
 	)
 
 	mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
-		Scheme:             scheme,
-		SyncPeriod:         &syncPeriod,
-		MetricsBindAddress: metricsAddr,
-		Port:               9443,
-		LeaderElection:     enableLeaderElection,
-		LeaderElectionID:   "appmesh-controller-leader-election",
+		Scheme:                 scheme,
+		SyncPeriod:             &syncPeriod,
+		MetricsBindAddress:     metricsAddr,
+		Port:                   9443,
+		LeaderElection:         enableLeaderElection,
+		LeaderElectionID:       "appmesh-controller-leader-election",
+		HealthProbeBindAddress: ":61779", // the liveness endpoint is default to "/healthz"
```

Comment:
Lets make this port configurable, via command line flag. We can use 61779 as the default.

Reply:
Sounds good. Updated the probe in config.

Test
```
$ curl http://192.168.85.101:61000/healthz
ok
```